Package.swift

```diff
@@ -6,10 +6,10 @@ import PackageDescription
 let package = Package(
     name: "Shout",
     products: [
-        .library(name: "Shout", targets: ["Shout"]),
+        .library(name: "Shout", type: .static, targets: ["Shout"]),
```

Comment:
What's the rationale for this change?

Reply:
Oh, that’s surprising, this has been pushed after my review. I’m strongly against this change though.

Reply:
Sorry, change I was trying on my fork that crept into the PR. I'll revert.